.changeset/beige-dots-film.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fluid-internal/test-end-to-end-tests": minor
+---
+
+end-to-end-tests package moved to internal scope
+
+The `@fluidframework/test-end-to-end-tests` packages has been renamed to @fluid-internal/test-end-to-end-tests and
+marked private.
```

Comment:
Let's just remove the part about "marked private" here since that didn't happen. @RishhiB I think you should be able to; if not, ping @tylerbutler.